lib/scripts/media.js

```diff
@@ -937,15 +938,18 @@ var dw_mediamanager = {
  * @param {string} opts
  * @param {string} align [none, left, center, right]
  */
-function dw_mediamanager_item_select(edid, mediaid, opts, align) {
+function dw_mediamanager_item_select(edid, mediaid, opts, align, keepopen) {
     var alignleft = '';
     var alignright = '';
     if (align !== '1') {
         alignleft = align === '2' ? '' : ' ';
         alignright = align === '4' ? '' : ' ';
     }
-
-    insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + '|', '}}', '');
+    if (keepopen) {
+        insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + '|}}', '', '');
```

Comment:
I think after the tag we can add `\n\n` for better composition. That is:
```js
insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + "|}}\n\n", '', '');
```

Reply:
I would disagree. I often use the feature to insert multiple images into one article but not necessarily one after each other. More like "one here, on there and another there..." having it insert new paragraphs would be counter productive then.

Reply:
@splitbrain is right. I ignored this use case before.